docs/source/index.rst

```diff
@@ -0,0 +1,107 @@
+.. PyDESeq2 documentation master file, created by
+   sphinx-quickstart on Thu Dec 22 16:34:24 2022.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to PyDESeq2's documentation!
+====================================
+
+This package is a python implementation of the `DESeq2 <https://bioconductor.org/packages/release/bioc/html/DESeq2.html>`_ method [1]_
+for differential expression analysis (DEA) with bulk RNA-seq data, originally in R.
+It aims to facilitate DEA experiments for python users.
+
+As PyDESeq2 is a re-implementation of `DESeq2 <https://bioconductor.org/packages/release/bioc/html/DESeq2.html>`_ from scratch, you may experience some differences in terms of retrieved values or available features.
+
+Currently, available features broadly correspond to the default settings of DESeq2 (v1.34.0) for single-factor analysis,
+but we plan to implement more in the near future. In case there is a feature you would particularly like to be implemented, feel free to open an issue on GitHub.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+ 
+   api/index
+
+Installation
+------------
+
+`PyDESeq2` can be installed from PyPI:
+
+.. code-block:: bash 
+   
+   pip install pydeseq2
+
+We recommend installing within a conda environment:
+
+.. code-block:: python
+
+   conda env create -n pydeseq2
+   conda activate pydeseq2
+   pip install pydeseq2
+
+
+If you're interested in contributing or want access to the development version, please see the :ref: contributing section.
+
+Requirements
+------------
+
+The list of package version requirements is available in  ``setup.py``, and will be automatically installed along PyDESeq2 when using PyPI.
+For reference, the code was tested with python 3.8 and the following package versions:
+
+- numpy 1.23.0
+- pandas 1.4.3
+- scikit-learn 1.1.1
+- scipy 1.8.1
+- statsmodels 0.13.2
+
+Please don't hesitate to open an issue in case you encounter any issue due to possible deprecations.
+
+Documentation
+-------------
+
+The documentation is automatically build using `Sphinx <https://www.sphinx-doc.org/en/master/>`_, and hosted `here on ReadTheDoc <https://pydeseq2.readthedocs.io/en/latest/>`_.
+
+Getting started
+---------------
+
+The `notebooks directory <https://github.com/owkin/PyDESeq2/blob/main/notebooks/README.md>`_ contains minimal examples on how to use PyDESeq2, in the form of jupyter notebooks.
+
+You can also try them from your browser (on synthetic data only):|Binder|
+
+.. |Binder| image:: https://mybinder.org/badge_logo.svg 
+   :target: https://mybinder.org/v2/gh/owkin/PyDESeq2/HEAD?labpath=notebooks%2Findex.ipynb
+
+Contributing
+------------
+
+PyDESeq2 is a living project and any contributions are welcome! The project is hosted on `Github <https://github.com/owkin/PyDESeq2>`_ .
```

Comment:
```suggestion
PyDESeq2 is a living project and any contributions are welcome! The project is hosted on `Github <https://github.com/owkin/PyDESeq2>`.
```

Reply:
I agree that the syntax is not really intuitive but it is the one of RST, the underscore is needed
See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages

Reply:
The problem is not the underscore but the whitespace between the underscore and the period